wqflask/base/trait.py

```diff
@@ -459,7 +540,7 @@ def retrieve_info(self, get_qtl_info=False):
                     logger.sql(query)
                     trait_qtl = g.db.execute(query).fetchone()
                     if trait_qtl:
-                        self.locus, self.lrs, self.pvalue, self.mean, self.additive= trait_qtl
+                        self.locus, self.lrs, self.pvalue, self.mean, self.additive = trait_qtl
```

Comment:
Initialization of local state (self.) here is pretty scary! Do we really need to maintain state in this object?